components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -137,6 +138,8 @@ func main() {
 	provisionAzureEventHub := provisioning.NewProvisionAzureEventHubStep(db.Operations(), azure.NewAzureProvider(), accountProvider, ctx)
 	runtimeStep := provisioning.NewCreateRuntimeStep(db.Operations(), db.Instances(), provisionerClient)
 	smOverrideStep := provisioning.NewServiceManagerOverridesStep(db.Operations(), cfg.ServiceManager)
+	//TODO: Uncomment when we azure client
```

Comment:
this comment is not valid as the line below is already uncommented

Reply:
done